### ISSUE TITLE
Should the repo be private?

### ISSUE THREAD

**Issue by mdb** [ericksteinberg/lilpopshop#1]
Hi Erick -- This repo is public. Should the repo be private?

Please let me know what you think.

Thanks,
Mike

**Comment by ericksteinberg** [comment]
Hey Mike,

I don't use github often and only have a free account. Would you want to make a private repo on your account and invite me as a collaborator? If not, just let me know and we can figure something out. 

Thanks!

## 

Erick Steinberg
www.JollyBureau.com

**Comment by mdb** [comment]
I don't pay for a private account either. Alternatively, BitBucket offers private repositories for free. I have one, though I am currently max'd out on the permitted number of collaborators. I suggest you sign up for BitBucket or coordinate with the client on covering GitHub repository hosting costs. Does that sound fair?

Thanks!

**Comment by ericksteinberg** [comment]
Hey Mike,

No problem! I'll go ahead and sign up for a bitbucket account and invite you to collaborate through there. 

Thanks for the help!

## 

Erick Steinberg
www.JollyBureau.com

**Comment by ericksteinberg** [comment]
Okay just added you on bitbucket, let me know if that worked. 

## 

Erick Steinberg

**Comment by mdb** [comment]
Yep - I pushed the repo. Please let me know if you have any problems.
